### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by PushDisDev**
PushDis project canceled and the server was already removed. Therefore, the client library doesn't work anymore.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @PushDisDev. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/08/30 13:54:42 Loaded 6143 libraries from DB
Removing PushDisConnector
Success!
```

**Pull request merged**
